R/bootstrap.R

```diff
@@ -77,22 +77,24 @@ bootstrapAUC <- function(SE_scored, annotationColName, signatureColNames,
          "Non-Boot AUC Values" = aucs))
 }
 
-#' Create a Table of Results for T-tests and Bootstrapped AUC for a Collection of Scored Signatures.
+#' Create a table of results for t-tests and bootstrapped AUCs for multiple scored signatures.
 #'
 #' This function collects the results of bootstrapping and t-tests for a scored
 #' gene expression dataset and presents them using a JavaScript table with an
-#' R interface.
+#' R interface, or as a \code{data.frame}.
 #'
 #' @inheritParams bootstrapAUC
+#' @param output a character string indicating the table output format. Possible
+#' values are \code{"DataTable"} and \code{data.frame}. The default is
+#' \code{DataTable}.
 #'
 #' @export
 #'
 #' @return A JavaScript table with an R interface using the \code{DT} package.
 #'
 #' @examples
 #' # Run signature profiling
-#'  choose_sigs <- subset(TBsignatures,
-#'                        !(names(TBsignatures) %in% c("Lee_4", "Roe_OD_4")))
+#'  choose_sigs <- TBsignatures[1:5]
```

Comment:
do we need to run 5 signatures in the example? we need to save time anywhere we can and examples don't have to be biologically meaningful, just show how the function works